components/api/src/main/java/com/hotels/styx/api/FullHttpRequest.java

```diff
@@ -106,7 +106,7 @@
      * Creates a request with the GET method.
      *
      * @param uri URI
-     * @return {@code this}
+     * @return {@link FullHttpRequest.Builder}
```

Comment:
The previous text was intended to show that it was the same builder being returned, so that chained calls can be used to set the values. The new text only tells us the type, which we already know.

Is there some builder convention we should be following?

Reply:
Even if the builder returned a new instance the method calls could still be chained.

We haven't agreed on any conventions. Feel free to suggest one.

Is this the only remaining issue that is blocking this PR?